data/modules/Combat/Combat.lua

```diff
@@ -523,7 +525,12 @@ local buildMissionDescription = function(mission)
 	})
 
 	desc.client = mission.client
-	desc.location = mission.location
+
+	if mission.status == "PENDING" and mission.rendezvous then
+		desc.location = mission.rendezvous
```

Comment:
```diff
-desc.location = mission.rendezvous
+desc.returnLocation = mission.rendezvous
```
I'm not sure if the only difference there is the message on the button. With **returnLocation** it's set to
`Set return route` instead of `Set destination as navigation target`

Reply:
Sorry, this is correct. There is no one clear return station.